sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SynchronousAnalyzer.java

```diff
@@ -21,19 +21,19 @@
 
 import com.google.common.util.concurrent.Uninterruptibles;
 import com.sonar.orchestrator.container.Server;
+import java.util.concurrent.TimeUnit;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Collections;
-import java.util.concurrent.TimeUnit;
-
 /**
  * See http://jira.sonarsource.com/browse/ORCH-263
  */
 public class SynchronousAnalyzer {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SynchronousAnalyzer.class);
-  static final String RELATIVE_URL = "/api/analysis_reports/is_queue_empty";
+
+
+  static final String RELATIVE_PATH = "/api/analysis_reports/is_queue_empty";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075)